mk/dev.mk

```
@@ -93,3 +95,7 @@ CIRCLECI_BADGE ?= [![CircleCI {{branch}}](https://img.shields.io/circleci/build/
 .PHONY: dev/repo-health
 dev/repo-health:
 	go run $(TOOLS_DIR)/dev/repo-health.go -action README -circleci-badge '$(CIRCLECI_BADGE)'
+
+.PHONY: dev/set-kuma-helm-repo
+dev/set-kuma-helm-repo:
+	${CI_TOOLS_BIN_DIR}/helm repo add kuma ${KUMA_CHARTS_URL}
```

```suggestion
	${CI_TOOLS_BIN_DIR}/helm repo add ${KUMA_CHART_REPO_NAME} ${KUMA_CHARTS_URL}
```

and a var somewhere?